backend/src/auth/service.py

```diff
@@ -37,12 +47,17 @@ def create_access_token(data, expires_delta=None):
                           If not provided, the token will expire in 15 minutes.
     :return: A JWT access token as a string.
     """
+    logger = Logger(__name__, "create_access_token").get_logger()
     to_encode = data.copy()
     if expires_delta:
         expire = datetime.utcnow() + expires_delta
     else:
-        expire = datetime.utcnow() + timedelta(minutes=Auth.DEFAULT_ACCESS_TOKEN_EXPIRE_MINUTES)
+        expire = datetime.utcnow() + timedelta(minutes=15)
```

Comment:
~~datetime.utcnow()~~
https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated
Use this (I think) instead
```python
datetime.now(timezone.utc)
```
also magic number?